lib/tasks/production-copy.rake

```diff
@@ -286,7 +284,7 @@ module CloneProductionHelper
     refs_to_clear += balanced_payments_refs_to_clear
     refs_to_clear += stripe_refs_to_clear
 
-    refs_to_clear.each do |model:,fields:|
+    refs_to_clear.each do |model, fields|
```

Comment:
this was a weird thing. I'm not sure how it worked before..

Reply:
confirmed that this fails.. I experimented with changing the names of the arguments to the block, I couldn't find a way to access said arguments.

Reply:
@ivar @thermistor removal of colons has broken `cleanse` for me (opposite of what Ivar found). Is this a difference of environment or was it a mistake while testing? I opened issue #3457 for this.